client/config/crd/groupsnapshot.storage.k8s.io_volumegroupsnapshotclasses.yaml

```diff
@@ -17,6 +17,15 @@ spec:
     - vgsclasses
     singular: volumegroupsnapshotclass
   scope: Cluster
+  conversion:
+    strategy: Webhook
+    webhook:
+      conversionReviewVersions: ["v1","v1beta1"]
+      clientConfig:
+        service:
+          namespace: default
+          name: snapshot-webhook-service
+          path: /convert
```

Comment:
If VGSClass v1beta1 is the same as v1beta2, I would not declare the conversion. The API server should do the right thing, without any code in the webhook

Reply:
Agree. Removed